dbms/src/DataStreams/ScanHashMapAfterProbeBlockInputStream.cpp

```diff
@@ -218,6 +218,12 @@ Block ScanHashMapAfterProbeBlockInputStream::readImpl()
             else
                 fillColumnsUsingCurrentPartition<false, false>(num_columns_left, columns_left, num_columns_right, columns_right, row_counter_column);
             break;
+        case ASTTableJoin::Kind::RightOuter:
```

Comment:
Looks like `case ASTTableJoin::Kind::RightOuter` and `case ASTTableJoin::Kind::RightAnti` is the same?

Reply:
Yeah, they can be merged.